examples/sandbox/.codesandbox/tasks.json

```diff
@@ -7,27 +7,19 @@
     },
     {
       "name": "Build App",
-      "command": "yarn build"
+      "command": "VITE_IS_SANDBOX=true yarn build"
     }
   ],
 
   // These tasks can be run from CodeSandbox. Running one will open a log in the app.
```

Comment:
`.json`, doesn't support comments. This would need to be a `.jsonc` file if we want to add comments. does codesandbox support `.jsonc`?

Reply:
Don't think so
https://codesandbox.io/docs/learn/devboxes/task

I'll remove comments then